apps/guardian-ui/src/AppContext.tsx

```diff
@@ -31,7 +33,6 @@ export interface AppContextProviderProps {
 export const AppContextProvider: React.FC<AppContextProviderProps> = ({
   children,
 }: AppContextProviderProps) => {
-  const api = new GuardianApi();
```

Comment:
This was causing every re-render of this context to create a new API instance. Bad times!

Reply:
cool memo